examples/person_search/models.py

```diff
@@ -0,0 +1,131 @@
+    "Market1501/bagtricks_R101-ibn.yml": "https://github.com/JDAI-CV/fast-reid/releases/download/v0.1.1/market_bot_R101-ibn.pth",
+    "Market1501/AGW_S50.yml": "https://github.com/JDAI-CV/fast-reid/releases/download/v0.1.1/market_agw_S50.pth",
+    "Market1501/AGW_R50.yml": "https://github.com/JDAI-CV/fast-reid/releases/download/v0.1.1/market_agw_R50.pth",
+    "Market1501/AGW_R50-ibn.yml": "https://github.com/JDAI-CV/fast-reid/releases/download/v0.1.1/market_agw_R50-ibn.pth",
+    "Market1501/AGW_R101-ibn.yml": "https://github.com/JDAI-CV/fast-reid/releases/download/v0.1.1/market_agw_R101-ibn.pth",
+    "Market1501/sbs_S50.yml": "https://github.com/JDAI-CV/fast-reid/releases/download/v0.1.1/market_sbs_S50.pth",
+    "Market1501/sbs_R50.yml": "https://github.com/JDAI-CV/fast-reid/releases/download/v0.1.1/market_sbs_R50.pth",
+    "Market1501/sbs_R50-ibn.yml": "https://github.com/JDAI-CV/fast-reid/releases/download/v0.1.1/market_sbs_R50-ibn.pth",
+    "Market1501/sbs_R101-ibn.yml": "https://github.com/JDAI-CV/fast-reid/releases/download/v0.1.1/market_sbs_R101-ibn.pth",
+    "Market1501/mgn_R50-ibn.yml": "https://github.com/JDAI-CV/fast-reid/releases/download/v0.1.1/market_mgn_R50-ibn.pth",
+
+    "DukeMTMC/bagtricks_S50.yml": "https://github.com/JDAI-CV/fast-reid/releases/download/v0.1.1/duke_bot_S50.pth",
+    "DukeMTMC/bagtricks_R50.yml": "https://github.com/JDAI-CV/fast-reid/releases/download/v0.1.1/duke_bot_R50.pth",
+    "DukeMTMC/bagtricks_R50-ibn.yml": "https://github.com/JDAI-CV/fast-reid/releases/download/v0.1.1/duke_bot_R50-ibn.pth",
+    "DukeMTMC/bagtricks_R101-ibn.yml": "https://github.com/JDAI-CV/fast-reid/releases/download/v0.1.1/duke_bot_R101-ibn.pth",
+    "DukeMTMC/AGW_S50.yml": "https://github.com/JDAI-CV/fast-reid/releases/download/v0.1.1/duke_agw_S50.pth",
+    "DukeMTMC/AGW_R50.yml": "https://github.com/JDAI-CV/fast-reid/releases/download/v0.1.1/duke_agw_R50.pth",
+    "DukeMTMC/AGW_R50-ibn.yml": "https://github.com/JDAI-CV/fast-reid/releases/download/v0.1.1/duke_agw_R50-ibn.pth",
+    "DukeMTMC/AGW_R101-ibn.yml": "https://github.com/JDAI-CV/fast-reid/releases/download/v0.1.1/duke_agw_R101-ibn.pth",
+    "DukeMTMC/sbs_S50.yml": "https://github.com/JDAI-CV/fast-reid/releases/download/v0.1.1/duke_sbs_S50.pth",
+    "DukeMTMC/sbs_R50.yml": "https://github.com/JDAI-CV/fast-reid/releases/download/v0.1.1/duke_sbs_R50.pth",
+    "DukeMTMC/sbs_R50-ibn.yml": "https://github.com/JDAI-CV/fast-reid/releases/download/v0.1.1/duke_sbs_R50-ibn.pth",
+    "DukeMTMC/sbs_R101-ibn.yml": "https://github.com/JDAI-CV/fast-reid/releases/download/v0.1.1/duke_sbs_R101-ibn.pth",
+    "DukeMTMC/mgn_R50-ibn.yml": "https://github.com/JDAI-CV/fast-reid/releases/download/v0.1.1/duke_mgn_R50-ibn.pth",
+}
+
+
+def _build_model(cfg, pretrained=True, verbose=True):
+    from torch.hub import (
+        _get_cache_or_reload,
+        load_state_dict_from_url
+    )
+
+    import importlib.util
+
+    repo_dir = _get_cache_or_reload(
```

Comment:
Two positional args, `trust_repo, calling_fn`, might be required. Looks like the additional arguments are introduced in Pytorch 1.12.

Maybe just call `load_state_dict_from_url` with `model_dir=None` and let Pytorch choose where to cache the model files?

Reply:
Thanks! I have updated the `models.py` from your suggestions:

1. To support the PyTorch 1.12, the `trust_repo` is set to be `check`, which will also be the default value after 1.14. 

2. The `model_dir` is set to None to enable PyTorch to decide the saving path.